rpm/install.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
```

Comment:
Why this file is outside of rpm-testing folder?

Reply:
I thought we wanted it to use it in other contexts like provisioning VMs not just Vagrant boxes. What should I do?

Reply:
Placing it inside rpm-testing seems good enough for me, as this way the vagrant vm is self contained, and the script is still reusable in other envs.

Reply:
Ok, new location is now `rpm-testing/install.sh`.